share/smack/top.py

```diff
@@ -420,13 +420,21 @@ def verify_bpl(args):
     command += ["/maxStaticLoopBound:%d" % args.loop_limit]
     command += ["/recursionBound:%d" % args.unroll]
 
+  elif args.verifier == 'symbooglix':
+    command = ['symbooglix']
+    command += [args.bpl_file]
+    command += ["--file-logging=0"]
+    command += ["--entry-points=%s" % ",".join(args.entry_points)]
+    command += ["--timeout=%d" % args.time_limit]
+    command += ["--max-depth=%d" % args.unroll]
```

Comment:
@zvonimir I would be interested in implementing it. Having thought about it some more there might be away to implement it that doesn't require all the tracing stuff to be implemented. I might give that approach a try as it will be a much faster to implement. Whilst I'm at it, do you need a procedure call bound as well?